Gemfile

```diff
@@ -26,6 +26,7 @@ end
 gem "puppet", :path => File.dirname(__FILE__), :require => false
 gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 2.0', '< 4'])
 gem "hiera", *location_for(ENV['HIERA_LOCATION'] || ['> 1.0', '< 3'])
```

Comment:
Not sure how to handle this dependency - the behavior in puppet now expects hiera to throw, so it must be a minimum of 2.0.0, but we have not released that yet...